cmd/zoekt-mirror-github/main.go

```diff
@@ -304,6 +304,8 @@ func cloneRepos(destDir string, repos []*github.Repository) error {
 			"zoekt.github-watchers":    itoa(r.WatchersCount),
 			"zoekt.github-subscribers": itoa(r.SubscribersCount),
 			"zoekt.github-forks":       itoa(r.ForksCount),
+
+			"zoekt.archived": marshalBool(*r.Archived),
```

Comment:
is it possible for this to be nil? If it is, I would suspect its for an old github enterprise API? If so, I'd default to archived being false if nil.

Reply:
I'm not sure whether it can be nil, so I've added the logic you've suggested (default to false if nil).